java-devcourse-project/src/HomeWorkApp.java

```diff
@@ -0,0 +1,93 @@
+public class HomeWorkApp {
+    public static void main(String[] args) {
+        printThreeWords("Orange", "Banana", "Apple");
+        checkSumSign();
+        printColor();
+        compareNumbers();
+        System.out.println(checkSumInRange(5, 16));
+        isPositiveOrNegative(1);
+        System.out.println(isPositiveOrNegativeBoolean(5));
+        printStrNTimes("str", 5);
+        System.out.println(isLeapYear(100));
+    }
+
+    static void printThreeWords(String firstWord, String secondWord, String thirdWord) {
+        System.out.println(firstWord);
+        System.out.println(secondWord);
+        System.out.println(thirdWord);
+    }
+
+    static void checkSumSign() {
+        int a, b;
+        a = 5;
+        b = -10;
+        int sum = a + b;
+        if (a + b >= 0) {
+            System.out.println("The sum of " + a + " and " + b + " = " + sum + " and it is positive");
+        } else {
+            System.out.println("The sum of " + a + " and " + b + " = " + sum + " and it is negative");
+        }
+    }
+
+    static void printColor() {
+        int value = 101;
+        if (value <= 0) {
+            System.out.println("RED");
+        } else if (value > 0 && value <= 100) {
```

Comment:
Это проверка уже выполняется выше и исключает текущую.
```suggestion
        } else if (value <= 100) {
```

Reply:
+